components/FullSiteArticleStats.js

```diff
@@ -5,16 +5,41 @@ import React from 'react';
  * @param {number} props.stats.repliedCount
  * @param {number} props.stats.notRepliedCount
```

Comment:
Please add a `@param {number} props.repliedArticleCount` here so that the doc string can be up-to-date :)